crates/sncast/src/starknet_commands/account/import.rs

```diff
@@ -139,3 +158,9 @@ fn get_private_key_from_file(file_path: &Utf8PathBuf) -> Result<Felt> {
     let private_key_string = std::fs::read_to_string(file_path.clone())?;
     Ok(private_key_string.parse()?)
 }
+
+fn get_private_key_from_input() -> Felt {
+    let private_key =
+        rpassword::prompt_password("Enter private key: ").expect("Failed to read private key");
+    Felt::try_from_hex_str(&private_key).expect("Failed to parse private key into Felt")
+}
```

Comment:
note: I didn't add test for entering private key interactively on purpose, because  `prompt_password` uses TTY and adding input with `.stdin("<private_key>")` will not work. Maybe I should add a test which is skipped when running all tests?